elasticdl/python/master/servicer.py

```diff
@@ -65,7 +65,7 @@ def __init__(
         if evaluation_service:
             evaluation_service.set_master_servicer(self)
 
-    # TODO: This is currently being used by multiple tests to initilize
+    # TODO: This is currently being used by multiple tests to initialize
```

Comment:
Who is `This`?

Reply:
`This` refers to function `set_model_var`. I am modifying this comment.